src/docs/repos/chrome/DOC.md

```diff
@@ -1,8 +1,8 @@
-# Insights Chrome
+# Chrome
```

Comment:
you removed 'Insights' here but not in the README?

Reply:
Now it's gone 😄